tools/release.js

```diff
@@ -131,55 +138,19 @@ async function startRelease() {
 
   // ---------------------------------------------------------------------------
 
-  log(`Step 7/${STEPS}: Creating the release...`);
+  log(`Step ${step++}/${STEPS}: Creating the release...`);
   const relaseResult = await execCommand(`gh release create v${next} dist/*.js dist/*.zip ${draft} -F ${file}`);
   if (relaseResult !== 0) {
     logError("github release failed. Aborting.");
     return;
   }
 
-  log(`Step 8/${STEPS}: publishing module on npm...`);
-  await execCommand("npm run publish");
+  log(`Step ${step++}/${STEPS}: publishing module on npm...`);
+  await execCommand("npm publish");
 
   log("Owl Release process completed! Thank you for your patience");
   await execCommand(`gh release view`);
   await execCommand(`gh release view -w`);
```

Comment:
probably need a step to copy last version of owl.iife.js into `docs/assets/owl.js`?

Reply:
There is one, except it's `owl.es.js` since I moved to es modules instead of using the owl global (which allows us to use import maps)